### PR TITLE
feat(anoncreds): add support for anoncreds issuance flow

### PR DESCRIPTION
**Pull request by goncalo-frade-iohk**
# Overview
Add support for anoncreds issuance

Fixes ATL-4294

### My PR contains...
* [ ] No code changes (changes to documentation, CI, metadata, etc.)
* [ ] Bug fixes (non-breaking change which fixes an issue)
* [ ] Improvements (misc. changes to existing features)
* [ ] Features (non-breaking change which adds functionality)

### My changes...
* [ ] are breaking changes
* [ ] are not breaking changes
* [ ] If yes to the above: I have updated the documentation accordingly

### Documentation
* [ ] My changes do not require a change to the project documentation
* [ ] My changes require a change to the project documentation
* [ ] If yes to the above: I have updated the documentation accordingly

### Tests
* [ ] My changes can not or do not need to be tested
* [ ] My changes can and should be tested by unit and/or integration tests
* [ ] If yes to the above: I have added tests to cover my changes
* [ ] If yes to the above: I have taken care to cover edge cases in my tests

### CODE REVIEW

File: AtalaPrismSDK/Pollux/Sources/PolluxImpl+CredentialRequest.swift [hunk 1]
```diff
@@ -17,18 +17,23 @@ extension PolluxImpl {
             throw PolluxError.offerDoesntProvideEnoughInformation
         }
         
-        switch offerAttachment.mediaType {
+        switch offerAttachment.format {
         case "jwt", "", .none:
```
Comment (curtis-h) [inline AtalaPrismSDK/Pollux/Sources/PolluxImpl+CredentialRequest.swift:21]: just checking has this been tested? I thought the jwt cred-offer had a format of "prism/jwt"

**Review by curtis-h — APPROVED** [review]

File: AtalaPrismSDK/Pollux/Sources/PolluxImpl+CredentialRequest.swift (hunk 1 above)
Comment (goncalo-frade-iohk) [reply AtalaPrismSDK/Pollux/Sources/PolluxImpl+CredentialRequest.swift]: This was a nice catch. I actually tested and worked on this morning but it didn't went on the commit my mistake. Updated now its solved ;)

**Review by cristianIOHK — APPROVED** [review]
Looks good
